mapflow/functional/service/data_catalog.py

```diff
@@ -61,13 +67,30 @@ def get_mosaic_callback(self, response: QNetworkReply):
         mosaic = MosaicReturnSchema.from_dict(response.readAll().data())
         self.mosaics.update({mosaic.id: mosaic})
         self.mosaicsUpdated.emit()
+    
+    def update_mosaic(self):  
+        mosaic = self.selected_mosaic()
+        dialog = UpdateMosaicDialog(self.dlg)
+        dialog.accepted.connect(lambda: self.api.update_mosaic(mosaic_id=mosaic.id, mosaic=dialog.mosaic(), callback=self.update_mosaic_callback))
+        dialog.setup(mosaic)
+        dialog.deleteLater()
+        self.dlg.mosaicTable.clearSelection()
+        self.dlg.mosaicInfo.setText("Mosaic info")
```

Comment:
Currently after updating mosaic, a row in a mosaicTable is deselected. 
Did this temporary because I had no luck changing it on a go: after updating it still shows old version of mosaic info (under the table) untill I re-click on it. 
Do you have any ideas on how I can change self.mosaics (so selected_mosaic() information changes as well) right after updating?